index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main">
+      <h2 id="title">Dr. Norman Borlaug</h2>
+      <div id="img-div">
+        <img id="image" />
+        <div id="img-caption">
+          Tribute to a man who saved over 2 million people from hunger
+        </div>
+      </div>
+      <section id="tribute-info">
+        <a id="tribute-link" href="https://google.com" target="_blank">
+          google information
+        </a>
+      </section>
+    </main>
+    <style>
+      #image {
+        display: block;
+        max-width: 100%;
+        height: auto;
+        margin: auto;
+      }
```

Comment:
Inline styling does not follow the correct gitflow, I think you should create an external CSS file.